src/track/taglib/trackmetadata_id3v2.cpp

```diff
@@ -593,7 +593,7 @@ bool importCoverImageFromTag(
 
     const TagLib::ID3v2::FrameList pFrames = iterAPIC->second;
     for (const auto coverArtType : kPreferredPictureTypes) {
-        for (const auto pFrame : pFrames) {
+        for (auto* const pFrame : pFrames) {
```

Comment:
This should become const auto& to make the best out of auto. Please verify other occurrences as well.

Reply:
Good catch. But it should become `const auto*` instead!

Reply:
...or even more restrictive `const auto* const`. The second const is optional and prevents manipulating the loop variable in the body.

Reply:
Did I mention that Rust saves me from writing such annoying boilerplate ;)

Reply:
We have to patters that works for any type as an universal solution: 
for (const auto& ..
for (auto&&

In other places we use ```for (const auto& .. `` for pointers as well. 
I don't mind if we go for explicit pointer syntax.

Reply:
Instead of using the `p` as a type prefix for pointers `auto*` is better. `auto*` can be verified by the compiler whereas `pSomething` is arbitrary and could be used for any type.

Reply:
It is just that the p Prefix is part of our style guide that's all. 

At least, this change does not alter the code and is fine in either way.